Source/Core/Common/MemoryUtil.cpp

```diff
@@ -88,20 +88,17 @@ void FreeMemoryPages(void* ptr, size_t size)
 {
   if (ptr)
   {
-    bool error_occurred = false;
-
 #ifdef _WIN32
     if (!VirtualFree(ptr, 0, MEM_RELEASE))
-      error_occurred = true;
+    {
```

Comment:
You should remove the braces you've added in this file (except the ones for the `mprotect` at the end, since the `if` there is two lines).